sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -7,10 +7,12 @@
 
 ### Breaking Changes
 - Renamed `begin_create_composed_model()` on `DocumentModelAdministrationClient` to `begin_compose_model()`.
-- Renamed `get_account_info()` on `DocumentModelAdministrationClient` to `get_resource_info()`.
-- Renamed `AccountInfo` model to `ResourceInfo`.
+- Renamed `get_account_info()` on `DocumentModelAdministrationClient` to `get_resource_details()`.
+- Renamed `AccountInfo` model to `ResourceDetails`.
```

Comment:
We have it as `ResourceDetails` too, but I think we may need to find a better name before GA (at least in .NET). Our architects complained about the name being too generic when we had it as `ResourceInfo`. `FormRecognizerResourceDetails` looks too long too.